tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -12,6 +12,7 @@
 class Contacts(Base):
 
     name = "Contacts"
+    entry_point = "contacts"
```

Comment:
Yes, this is used when the contacts app gets launch. Otherwise the communications url is launched, which then shows a directory listing (you can see the same happening with webIDE).